src/main/groovy/edu/oregonstate/mist/api/BuildInfoManager.groovy

```diff
@@ -0,0 +1,33 @@
+package edu.oregonstate.mist.api
+
+import io.dropwizard.lifecycle.Managed
+
+/**
+ * Created by georgecrary on 7/26/16.
+ */
+class BuildInfoManager implements Managed {
+    Info getInfo() {
+        info
+    }
+    private Info info = new Info()
+
+    @Override
+    public void start() throws Exception {
+        def stream = InfoResource.class.getResourceAsStream('/build.properties')
```

Comment:
Okay, I see the changes, but can we do 
def stream = this.class.getResourceAsStream('/build.properties') in BuildInfoManager.groovy, I wonder why you call it from InfoResource when you are at BuildInfoManager?